
float32 roll_rate_integ	    # roll rate inegrator
float32 pitch_rate_integ	# pitch rate integrator
float32 yaw_rate_integ	    # yaw rate integrator
float32 raw_derivative
float32 rollrate_foaw_derivative
float32 pitchrate_foaw_derivative
float32 yawrate_foaw_derivative
float32 rates_lpf
float32 raw_derivative_rates_lpf
float32 foaw_gyro_lpf
float32 dt
uint8 window_size
